Ghidra/Processors/6800/data/languages/6800.slaspec

```diff
@@ -0,0 +1,459 @@
+        local tmpResult = reg + op;
+        setHFlag(reg, op);
+        $(C) = carry(reg, op) || carry(tmpResult, carryIn);
+        $(V) = scarry(reg, op) ^^ scarry(tmpResult, carryIn);
+        reg = tmpResult + carryIn;
+        setNZFlags(reg);
+}
+
+macro subtraction(reg, op) {
+        $(V) = sborrow(reg, op);
+        reg = reg - op;
+        setNZFlags(reg);
+        $(C) = (reg < op);
+}
+
+macro subtractionWithCarry(reg, op) {
+        local carryIn = $(C);
+        local tmpResult = reg - op;
+        $(C) = (reg < op) || (tmpResult < carryIn);
+        $(V) = sborrow(reg, op) ^^ sborrow(tmpResult, carryIn);
+        reg = tmpResult - carryIn;
+        setNZFlags(reg);
+}
+
+macro compare(reg, op) {
+        $(V) = sborrow(reg, op);
+        local tmp = reg - op;
+        setNZFlags(tmp);
+        $(C) = (tmp < op);
+}
+
+macro compare16(reg, op) {		# used for 16 bit CPX; C not affected
+        $(V) = sborrow(reg, op);
+        local tmp = reg - op;
+        setNZFlags(tmp);
+}
+
+macro logicalAnd(reg, op) {
+        reg = reg & op;
+        setNZFlags(reg);
+        $(V) = 0;
+}
+
+macro logicalOr(reg, op) {
+        reg = reg | op;
+        setNZFlags(reg);
+        $(V) = 0;
+}
+
+macro logicalExclusiveOr(reg, op) {
+        reg = reg ^ op;
+        setNZFlags(reg);
+        $(V) = 0;
+}
+
+macro bitTest(reg, op) {
+        local tmp = reg & op;
+        setNZFlags(tmp);
+        $(V) = 0;
+}
+
+macro loadRegister(reg, op) {
+        reg = op;
+        setNZFlags(reg);
+        $(V) = 0;
+}
+
+macro storeRegister(reg, op) {
+        op = reg;
+        setNZFlags(reg);
+        $(V) = 0;
+}
+
+macro Push1(reg, op) {		# 1 byte operand
+        *:1 reg = op;
+        reg = reg - 1;
+}
+
+macro Push2(reg, op) {		# 2 byte operand
+        reg = reg - 1;
+        *:2 reg = op;
+        reg = reg - 1;
+}
+
+macro Pull1(reg, op) {
+        op = *:1 reg;
+        reg = reg + 1;
+}
+
+macro Pull2(reg, op) {
+        reg = reg + 1;
+        op = *:2 reg;
+        reg = reg + 1;
+}
+macro PushEntireState() {
+		local addr:2 = inst_next + 1; 
+		Push2(SP, addr);
+		Push2(SP, X);	
+        Push1(SP, A); 
+        Push1(SP, B); 
+        Push1(SP, CC);
+}
+macro PullEntireState() {
+		local addr:2; 
+		Pull1(SP, CC); 
+		Pull1(SP, B);
+        Pull1(SP, A); 
+        Pull2(SP, X); 
+        Pull2(SP, addr); 
+        return [addr];
+}
+
+################################################################
+# Addressing tables
+################################################################
+
+#	Branches and jumps addresses
+OPREL: addr     is simm8  [ addr = inst_start + simm8 + 2;  ]   { export *:2 addr; } 			# relative
+OPJ: imm8",X" 	is (ophi=0x6 | ophi=0xA); imm8 { addr:2 = X + imm8; export *:2 addr; } 			# indexed
+OPJ: imm16 		is (ophi=0x7 | ophi=0xB); imm16 { export *:2 imm16; } 							# extended
+
+#	OPAB  = operand for operations on accumulators A or B
+OPAB: "#"^imm8  is (ophi=0x8 | ophi=0xC); imm8 { export imm8; } 								# immediate
```

Comment:
just being bit by issues that look like this I would `export imm8:1`

Reply:
same

Reply:
try `export *[const]:1 imm8;`